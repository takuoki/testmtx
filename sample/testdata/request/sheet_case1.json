{
  "num_key": 101,
  "string_key": "string value 101",
  "bool_key": true,
  "object_key": {
    "key1": 201,
    "key2": "string value 201"
  },
  "array_key": [
    {
      "key3": 301,
      "key4": "string value 301"
    },
    {
      "key3": 401,
      "key4": "string value 401"
    }
  ]
}