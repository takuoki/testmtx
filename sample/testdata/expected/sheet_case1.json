{
  "status": "success",
  "code": 200
}